libraries/botbuilder-dialogs-adaptive/src/converters/languageGeneratorConverter.ts

```diff
@@ -6,10 +6,11 @@
  * Licensed under the MIT License.
  */
 
-import { Converter } from 'botbuilder-dialogs-declarative';
+import { Converter } from 'botbuilder-dialogs';
 import { ResourceMultiLanguageGenerator } from '../generators';
 
-export class LanguageGeneratorConverter implements Converter {
+export class LanguageGeneratorConverter
+    implements Converter<string | ResourceMultiLanguageGenerator, ResourceMultiLanguageGenerator> {
```

Comment:
We could probably push down the `| T` piece to the underlying `Converter` type.

```typescript
type Converter<From, To> = (value: From | To) => To;
```

Then you wouldn't need to always include `| T` everywhere. Not sure if this will break backwards compat with any of the existing converters though.

Reply:
We might need to update all existing converters to support "convert T to T".